src/ToolServiceProvider.php

```diff
@@ -3,6 +3,10 @@
 namespace Tipoff\StaffBookings;
 
 use Illuminate\Support\ServiceProvider;
+use Illuminate\Support\Facades\Route;
+use Laravel\Nova\Events\ServingNova;
+use Laravel\Nova\Nova;
+use Tger\Booking\Http\Middleware\Authorize;
 
 class ToolServiceProvider extends ServiceProvider
```

Comment:
I guess namespace should fit the package name.

Reply:
Good catch - missed that one. I've fixed and tested that issue and pushed up the fix commit to this branch.